src/Library/demos/Text Input/main.blp

```diff
@@ -0,0 +1,309 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.Clamp {
+  Box {
+    orientation: vertical;
+
+    Label {
+      label: "Text Input";
```

Comment:
I think I like "Text Fields" better - it's what HIG uses https://developer.gnome.org/hig/patterns/controls/text-fields.htmlhttps://developer.gnome.org/hig/patterns/controls/text-fields.html

There are other widgets for text input so it's a bit too generic here.

Reply:
Okay I'll change that